src/Products/Annotation/Controllers/AnnotationApiController.cs

```diff
@@ -271,159 +241,45 @@ public HttpResponseMessage LoadDocumentPage(AnnotationPostedDataEntity loadDocum
             }
         }
 
-        /// <summary>
-        /// Download document
-        /// </summary>
-        /// <param name="path">string</param>
-        /// <param name="annotated">bool</param>
-        /// <returns></returns>
-        [HttpGet]
-        [Route("downloadDocument")]
-        public HttpResponseMessage DownloadDocument(string path)
+        private List<string> GetAllPagesContent(string password, string documentGuid, IDocumentInfo pages)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetAllPagesContent' a static method.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)